_scala_carousel_items/1-collections-ops.md

```diff
@@ -0,0 +1,20 @@
+---
+optionId: collections-ops
+scastieLink: 'https://scastie.scala-lang.org/IRGiFXDhQPeOXYsW41FnCw'
+codeTitle: 'Functional programming with immutable collections'
+description: "High-level operations avoid the need for complex and error-prone loops."
+---
+
+```scala
+val fruits =
+  List("apple", "banana", "avocado", "papaya")
+
+val countsToFruits = // count how many 'a' in each fruit
+  fruits.groupBy(f => f.count(_ == 'a'))
```

Comment:
```suggestion
  fruits.groupBy(fruit => fruit.count(_ == 'a'))
```

Reply:
oh weird I thought I already had this change

Reply:
added